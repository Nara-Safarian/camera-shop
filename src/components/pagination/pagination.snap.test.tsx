import { render } from '@testing-library/react';
import Pagination from './pagination';

test('Pagination snapshot should be rendered correctly', () => {
  const {container} = render(
    <Pagination activePage={2} pagesCount={10} onClickNextPage={function (): void {
      // nothing
    } } onClickPrevPage={function (): void {
      // nothing
    } } onClickPage={function (): void {
      // nothing
    } }
    />
  );
  expect(container).toMatchSnapshot();
});
