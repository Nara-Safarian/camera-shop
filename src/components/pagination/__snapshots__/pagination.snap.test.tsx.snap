// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination snapshot should be rendered correctly 1`] = `
<div>
  <div
    class="pagination"
  >
    <ul
      class="pagination__list"
    >
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link pagination__link--text"
          href="catalog/page_1"
        >
          Назад
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_1"
        >
          1
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link pagination__link--active"
          href="catalog/page_2"
        >
          2
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_3"
        >
          3
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_4"
        >
          4
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_5"
        >
          5
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_6"
        >
          6
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_7"
        >
          7
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_8"
        >
          8
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_9"
        >
          9
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link "
          href="catalog/page_10"
        >
          10
        </a>
      </li>
      <li
        class="pagination__item"
      >
        <a
          class="pagination__link pagination__link--text"
          href="catalog/page_3"
        >
          Далее
        </a>
      </li>
    </ul>
  </div>
</div>
`;
